frontend/src/pages/Setup/SetupRouter/AlertsSetup.tsx

```diff
@@ -515,25 +518,65 @@ const AlertPicker = function ({
 				)
 			})}
 
-			<Box
-				mt="8"
-				alignItems="center"
-				display="flex"
-				gap="4"
-				color="weak"
-				flexWrap="nowrap"
-			>
-				<Text size="medium" color="moderate" lines="1">
-					We've prefilled the alert configurations based on typical
-					use-cases.{' '}
-				</Text>
-				<TextLink href="/alerts">
-					<Box alignItems="center" display="flex" gap="2">
-						<Text size="medium">Go to alerts</Text>
-						<IconSolidCheveronRight />
-					</Box>
-				</TextLink>
-			</Box>
+			<Stack gap="4" mt="24">
+				<Box>
+					<Button
+						trackingId="save-alerts"
+						kind="primary"
+						size="small"
+						emphasis="high"
+						iconRight={<IconSolidCheveronRight />}
+						loading={createLoading.current}
+						disabled={createLoading.current}
+						onClick={async () => {
+							try {
+								createLoading.current = true
+								await createAlerts()
+								message.success(
+									`${alertsSelected.length} alert${
+										alertsSelected.length > 1 ? 's' : ''
+									} updated!`,
+								)
+								await new Promise((r) => {
```

Comment:
We should have a utility function named `wait`, cause this is a very common thing to do.

Reply:
yeah good point! dropping this change tho per feedback from today's demo session.